src/integration-test/java/com/mongodb/jdbc/integration/MongoSQLIntegrationTest.java

```diff
@@ -55,4 +77,70 @@ public void loadTestConfigs() throws IOException {
         }
         return dynamicTests;
     }
+
+    @Test
+    public void testTypesOther() throws SQLException, IOException, IllegalAccessException {
+        DataLoader loader = new DataLoader(TEST_DATA_OTHER_DIRECTORY);
+        loader.dropCollections();
+
+        Map<String, Object> row = new LinkedHashMap<>();
+        ObjectId objID = new ObjectId("000000000000000000000001");
+        BsonObjectId objectID = new BsonObjectId(objID);
+        BsonDbPointer dbPointer = new BsonDbPointer("namespace", objID);
+        BsonJavaScript javaScript = new BsonJavaScript("function(){ }");
+        BsonRegularExpression regularExpression = new BsonRegularExpression("a(bc)*");
+        BsonSymbol symbol = new BsonSymbol("symbol");
+        BsonTimestamp timestamp = new BsonTimestamp(100);
+
```

Comment:
We also use `Types.OTHER` for the bson types `object`, `undefined`, `symbol`, and `javascriptWithScope`. I know the latter 3 are deprecated but I believe they should still be available in the driver so we may still want to test them. At minimum, we should test an object.

Reply:
Sounds good.  `symbol` is there currently.  I missed `javascriptWithScope`, I'll add that.

I'm having trouble with `object`, I am inserting a `BasicBSONObject` and I get back a `BsonDocument` so the comparison fails.  I'll see if I can cast it somehow.
The `undefined` type is throwing this error: `'failed getting result set schema: failed to add JSON schema for collection integration_test.types_other to the catalog: is not a valid BSON type`.  I'll check in the mongosql-rs if there's a bug.

Reply:
- Ah, I missed the `symbol`. 
- For object, I think putting in a BsonDocument as the input type is fine since that is also logically an object
- Oh yea, we have a bug ticket in mongosql-rs to include the undefined type ([SQL-395](https://jira.mongodb.org/browse/SQL-395))

Reply:
Added `object` and `javascriptWithScope`.  For `undefined` I added the bug ticket in a comment.